.circleci/config.yml

```diff
@@ -721,6 +730,18 @@ jobs:
             core0.test_hello_argc
             core2.test_demangle_stacks_symbol_map"
       - upload-test-results
+  test-rust:
+    executor: linux-python
```

Comment:
Another option here might be the CircleCI rust image? https://circleci.com/developer/images/image/cimg/rust

Reply:
I tried that first but decided against it for 2 reasons:

1. We need python to run emscripten and the rust image does not have python installed.
2. We want to run rust nightly I think, which the rust image doesn't seem to have

Reply:
Ah, interesting. Are there changes going into core rustc that we will need?

Reply:
As an aside I find it odd that circle CI doesn't have base images with more than one tool.  Surely there are a lot of polyglot projects out there that needs python + node or python + rust.   It seems that such projects currently have to build their own base image or manually install one or more runtimes.

Reply:
What's the recommended way to do that?

Reply:
`rustup toolchain install nightly`

Looks like you might also be able to specify it when you install rustup https://rust-lang.github.io/rustup/installation/index.html#installing-nightly